test/transforms.jl

```diff
@@ -41,4 +41,24 @@
     @test nelements(smesh) == nelements(mesh)
     @test topology(smesh) == topology(mesh)
   end
+
+  @testset "Rotation" begin
+    # a rotation doesn't change the topology
+    trans = Rotation(EulerAngleAxis(pi/4, [1, 0, 0]))
+    @test TB.isrevertible(trans)
+    mesh  = readply(T, joinpath(datadir,"beethoven.ply"))
+    rmesh = trans(mesh)
+    @test nvertices(rmesh) == nvertices(mesh)
+    @test nelements(rmesh) == nelements(mesh)
+    @test topology(rmesh) == topology(mesh)
+
+    # check rotation on a triangle
+    triangle = Triangle(Point(0,0,0), Point(1,0,0), Point(0,1,0))
+    trans = Rotation(EulerAngleAxis(-pi/2, [0, 0, 1]))
+    rtriangle = trans(triangle)
+    @test in(Point(0,0,0), rtriangle)
+    @test in(Point(-0.01,0.9,0), rtriangle)
+    @test in(Point(-0.9,0.01,0), rtriangle)
```

Comment:
I think the right test here is to check that the vertices(rtriangle) are equal to specific points.

Reply:
You can do `@test p1 == p2` and the test will use approximate equality already implemented for `Point`